### DISCUSSION TITLE
how to invert Axis?

### DISCUSSION THREAD

**Discussion by ikikaidesu** [Live-Charts/LiveCharts2#1534] [Q&A] [answered]
Hi, i wanna to invert my axis(from up 0 to down 100), i read documentation and fount this([https://livecharts.dev/docs/Eto/2.0.0-rc2/CartesianChart.Axes%20properties#inverted-property](url)), but when i try to write it in axis setting the error appears. Can someone pls tell me what i do wrong?
![image](https://github.com/beto-rodriguez/LiveCharts2/assets/124306079/2c8c57de-7fee-4064-bdbe-397de0ee7b2d)

there is full axisY code
![image](https://github.com/beto-rodriguez/LiveCharts2/assets/124306079/5e49e8b0-e523-4dbd-ad28-6ed2fef2fd2c)

**Comment by igiona** [comment, marked as answer]
I was able to to do so in the view-model, where I create the data series:

Viewmodel
```c#
...

XAxes = [
    new Axis { 
        Labels = xAxisLabels
    }
];
YAxes = [
    new Axis {
        Labels = yAxisLabels,
        IsInverted = true,
        MinStep = 1
    }
];
```
View
```xaml

      <lvc:CartesianChart
            Height="600"
            Width="1200"
            Series="{Binding MySeries}"
            XAxes="{Binding XAxes}"
            YAxes="{Binding YAxes}">
```
